build/archives/BUILD.gn

```diff
@@ -74,6 +79,19 @@ if (build_engine_artifacts) {
         destination = "libtessellator$dll"
       },
     ]
+    if (host_os == "mac" && full_target_platform_name == "darwin-x64") {
+      if (flutter_runtime_mode == "debug") {
+        file_with_entitlements = [
+          "flutter_tester",
```

Comment:
> I'm more interested in how we chose this set of files. Is it only the files that have an executable bit set? if so, we can automate searching the generated directory for the executable files instead of hard coding them.

macOS binaries that run on the host with gatekeeper need entitlements. iOS target binaries must not have entitlements, because when we build the iOS app, Xcode will sign them, and I believe if a binary already has entitlements and it gets resigned with different entitlements bad things happen?